core/src/sigverifier/bls_sigverifier.rs

```diff
@@ -4,13 +4,27 @@
 use {
     crate::sigverify_stage::{SigVerifier, SigVerifyServiceError},
     alpenglow_vote::bls_message::BLSMessage,
-    crossbeam_channel::Sender,
+    crossbeam_channel::{Sender, TrySendError},
     solana_streamer::packet::PacketBatch,
+    std::time::{Duration, Instant},
 };
 
-#[derive(Default, Clone)]
+const STATS_INTERVAL_SECONDS: u64 = 10; // Log stats every 10 second
+
+// We are adding our own stats because we do BLS decoding in batch verification,
+// and we send one BLS message at a time. So it makes sense to have finer-grained stats
+#[derive(Debug, Default, Clone)]
+pub(crate) struct BLSSigVerifierStats {
+    pub bls_messages_sent: u64,
+    pub bls_messages_malformed: u64,
+    pub sent_failed: u64,
+    pub packets_received: u64,
+}
+
 pub struct BLSSigVerifier {
-    sender: Option<Sender<BLSMessage>>,
+    sender: Sender<BLSMessage>,
+    stats: BLSSigVerifierStats,
+    last_stats_logged: Instant,
```

Comment:
I think it would be better if this was part of the `BLSSigVerifierStats` struct

Reply:
Thanks for the review, since this one is already merged, I'll address your comments in https://github.com/anza-xyz/alpenglow/pull/210

Reply:
Yeah this is me being lazy and wanted to use Default(), changed.